Resources/public/javascript/forms.js

```diff
@@ -1,16 +1,61 @@
+let removePopup = `<div class="sonata-ba-delete popup-file-delete mfp-hide">
+                        <div class="box box-danger">
+                            <div class="box-header">
+                                <h3 class="box-title">Bevestig verwijdering</h3>
+                            </div>
+                            <div class="box-body">
+                                Weet je zeker dat de geselecteerde afbeelding moet worden verwijderd?
+                            </div>
+                            <div class="box-footer clearfix">
+                                <button class="btn btn-danger js-trigger-save"><i class="fa fa-trash" aria-hidden="true"></i> Ja, verwijder</button>
+                                <button class="btn btn-success js-trigger-cancel" href="/admin/wearejust/example/post/1/edit">
+                                <i class="fa fa-pencil" aria-hidden="true"></i>
+                                Nee, annuleer</button>
+                            </div>
+                        </div>
+                        <button title="Close (Esc)" type="button" class="mfp-close js-trigger-cancel">×</button>
+                        <div class="anim-loading">
+                            <label></label>
+                            <label></label>
+                            <label></label>
+                            <label></label>
+                        </div>
+                    </div>`;
+
 class imageRemove {
     constructor(item) {
+        $('body').append(removePopup);
```

Comment:
Why not put the HTML in the views? Adding HTML to JS is a bad idea IMHO. It can cause glitches when loading the page, frontenders have no clue where to look when wanting to change something, code completion doesnt work and above all it looks very hacky.